targets/targets.mk

```diff
@@ -10,12 +10,20 @@ $(eval $(call GluonTarget,x86,geode))
 $(eval $(call GluonTarget,x86,64))
 
 ifneq ($(BROKEN),)
+  AP_11S=1
```

Comment:
- the indention of this variable is different from the other lines
- the naming might introduce conflicts. maybe: GLUON_TARGET_11S_ONLY

Reply:
ACK